File: example_bots/java/BluffBot.java
/**
 * A bot that always bluffs
 *
 * This bot will always raise the maximum amount (going all in if possible)
 */
public class BluffBot  {
   public static final String BOT_NAME = "Bluff bot";

   public static int act(Observable obs) throws Exception {

      return obs.getMaxRaise();
   }
 }

File: example_bots/python/bluff_bot.py
from poker_game_runner.state import Observation
from poker_game_runner.utils import Range, HandType
"""
  This bot will always raise the pot with it's entire cash stack (going "all-in")
"""


class Bot:
    def get_name(self):
        return "bluff_bot"

    def act(self, obs: Observation):
        return obs.get_max_raise()
